interfaces/edit_volunteer_cli.py

```diff
@@ -0,0 +1,79 @@
+from models.volunteer import Volunteer
+from models.camp import Camp
+from interfaces.admin_cli import ManageVolunteerMenu
+
+
+###---- Manage Volunteer Menu ----
+class EditVolunteerMenu(ManageVolunteerMenu):
+    def __init__(self):
+        super(EditVolunteerMenu, self).__init__()
+
+    def edit_volunteer_menu(self) -> None:
+        print("""
+         Select an item you'd like to edit:
+         --------
+         1) Firstname
+         2) Lastname
+         3) Phone number
+         4) Camp
+         5) Availability
+
+         R) Return to previous page
+         0) Log-out
+         """)
+
+    def precmd(self, option: str) -> int:
+        edit_volunteer_options = {"logout": 0,
+                                  "edit_firstname": 1,
+                                  "edit_lastname": 2,
+                                  "edit_phone": 3,
+                                  "edit_camp": 4,
+                                  "edit_availability": 5,
+                                  "return": "r"}
+
+        ### Add input error handling
+        if option.isdigit():
```

Comment:
Needs some 'and' condition or 2nd 'if' loop to make sure that the digit specified is in range e.g. something like

```
if option.isdigit():
    if 0 <= x <= 5:
         return list(edit_volunteer_options.keys())[list(edit_volunteer_options.values()).index(int(option))]
```
x being a placeholder for a better variable.

Reply:
Good suggestion! I added a specified range in the newly pushed version.